app.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "Gunstore.io",
+    "description": "A Node.js app using Express and gun",
+    "repository": "https://github.com/i001962/gunstore.io",
```

Comment:
this points to your fork and should be corrected

Reply:
Oops. Will change and revert.